.prettierrc

```
@@ -0,0 +1,6 @@
+{
+	"trailingComma": "es5",
+	"tabWidth": 4,
+	"semi": false,
+	"singleQuote": true
+}
```

adapted from the live site:
```suggestion
{
  "printWidth": 100,
  "jsxBracketSameLine": false,
  "singleQuote": true,
  "tabWidth": 2,
  "trailingComma": "es5",
  "semi": false,
  "plugins": ["prettier-plugin-prisma"],
}
```